test/mocks/server/mocks.h

```diff
@@ -135,6 +135,15 @@ class MockHotRestart : public HotRestart {
   MOCK_METHOD0(terminateParent, void());
   MOCK_METHOD0(shutdown, void());
   MOCK_METHOD0(version, std::string());
+
+  Thread::BasicLockable& logLock() override { return log_lock_; }
```

Comment:
nit: These should be normal mock definitions with ON_CALL defaults to return the members.

Reply:
I'll tackle this next.